worker/worker.go

```diff
@@ -493,4 +498,45 @@ func (w *Worker) searchForDanglingIssues(repository *models.Repository) {
 	}
 }
 
+/* Deletes cache issues and issue-count cached data. Then cache the default issues of the issues landing page */
+func deleteAndUpdateCache() {
+	cacheConn := cache.CachePool.Get()
+	defer cacheConn.Close()
+
+	cache.DeleteKeysByPattern(&cacheConn, "issues:*")
+	cache.DeleteKeysByPattern(&cacheConn, "issues-count:*")
+
+	tx, err := models.DB.NewTransaction()
+	defer tx.Close()
+
+	if err != nil {
+		fmt.Println(errors.New("no transaction found"))
+		return
+	}
+
+	params := url.Values{}
+	issues := &models.Issues{}
+	defaultIssuesWhereClause := "issues:closed = false"
+	cacheKey := "issues:" + defaultIssuesWhereClause + "and page=1"
+	for _, filter := range []string{"language", "experience_needed", "type", "project_id", "ordering"} {
+		params.Set(filter, "undefined")
+	}
+	params.Set("page", "1")
+	q := tx.PaginateFromParams(params).Eager()
+	ok, _ := cache.Exists(&cacheConn, cacheKey)
+
+	if !ok {
+		if err := q.Where(defaultIssuesWhereClause).All(issues); err != nil {
+			fmt.Println(errors.WithMessage(err, "DB Operation falied"))
+			return
+		}
+
+		jsonIssues, _ := json.Marshal(issues)
```

Comment:
could you print the error here and return? If the issues could be marshalled they are not going to be parsed properly when a client tries to get them